src/Tests/SNIPPETS.cs

```diff
@@ -207,6 +207,15 @@ operation DependsOnChemistryWorkspace() : ((JordanWignerEncodingData, Int, Doubl
     }
 ";
 
+        public static string InvalidEntryPoint =
+@"
+    /// # Summary
+    ///     This script has an operation that is not valid to be marked as an entry point.
```

Comment:
```suggestion
    /// This script has an operation that is not valid to be marked as an entry point.
```

Reply:
Leaving this as-is for readability, since it's consistent with the rest of the file.